spec/labor/profile_image_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe ProfileImage, type: :labor do
+  let(:user) { create(:user) }
+
+  it "returns a profile_image" do
+    expect(described_class.new(user).get.size).to be > 0
+  end
+
+  it "renders an error when initializing with a nil object" do
+    expect { described_class.new(nil) }.to raise_error(NoMethodError)
+  end
```

Comment:
The reason why your test pass is because you're testing there is a bug, instead of testing that there isn't :D

Reply:
hahaha it make sense now